test/plotrecipes.jl

```diff
@@ -33,4 +33,10 @@
       @test_ref_plot "data/hscatter.png" plt
     end
   end
+
+  @testset "EmpiricalHistogram" begin
+    if visualtests
+      @test_ref_plot "data/empiricalhistogram.png" plot(EmpiricalHistogram(d, :z₁))
+    end
+  end
```

Comment:
Please move this testset to a separate file called `histograms.jl` and feel free to add more tests with different distributions. :)

Reply:
Sure!